src/styles/block/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    height: 134px;
+  }
+
+  &__description {
+    margin: 40px 0 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+    text-align: left;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  &__reviews {
+    margin: 16px 0 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-total {
+      margin: 0;
+      color: $main-text-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 0 0;
+
+    &-label {
+      font-size: 12px;
+      color: $secondary-text-color;
+      margin: 0;
+    }
+
+    &-sum {
+      line-height: 18px;
+      font-size: 16px;
+      font-weight: 700;
+      color: $main-text-color;
+      margin: 0;
+    }
```

Comment:
for a modifier you need to use `--` if its not modifier not use `&-` write all selectors name for example ( `&__price-label, &__price-sum`) for better searching slectors in feature.